.travis.yml

```diff
@@ -16,7 +16,7 @@ before_install:
   - sudo apt-get -y install shunit2
 script:
     - cd linux/test
-    - ./docker-build.sh $ENV
+    - OMEROVER=OMERO-DEV-latest ./docker-build.sh $ENV
```

Comment:
only to have travis to pass otherwise m9 will be used with contain omero-marshal dependencies installed using git